src/test/java/dao/CustomerDAOTest.java

```diff
@@ -1,4 +1,93 @@
 package dao;
 
+import entities.Customer;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import services.ConnectionFactory;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.util.Set;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
 public class CustomerDAOTest {
+    private static final int NUMBER_OF_ENTRIES = 10;
+
+    @BeforeClass
+    public static void setupDatabase() throws SQLException {
+        Connection connection = ConnectionFactory.getConnection();
+        for (int i = 0; i < NUMBER_OF_ENTRIES; i++) {
+            PreparedStatement preparedStatement =
+                    connection.prepareStatement("INSERT INTO Customer (Name, LastName, PersonID) VALUES(?, ?, ?)");
+            preparedStatement.setString(1, "NAME_ENTRY#" + i);
+            preparedStatement.setString(2, "LASTNAME_ENTRY#" + i);
+            preparedStatement.setString(3, "PERSONID_ENTRY#" + i);
+            preparedStatement.executeUpdate();
+        }
```

Comment:
this is match better, you can also use your dao insert (and delete in after case) method to do it actually